docs/administrators/authorization.rst

```diff
@@ -0,0 +1,275 @@
+* Environment-scoped roles are defined in the ``urn:inmanta:roles`` claim of the access token. The value must be a dictionary that maps the ID of the environment to a list of roles the user has in that environment.
+* Global admins must have the claim ``urn:inmanta:is_admin`` in the access token with the value set to ``true``.
+
+
+Integration with database authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default policy integrates seamlessly with database authentication.
+
+* The admin user created using the ``/opt/inmanta/bin/inmanta-initial-user-setup`` command will have the ``urn:inmanta:is_admin`` claim set to true.
+* The web-console provides support to manage users and role assignments.
+
+If the policy contains a ``roles`` variable that contains a list role names, the Inmanta server will create these roles automatically when the server starts. The default policy defines this variable for the roles it can handle. Like that there is no need to create roles by hand. The snippet below provides an example. Removing a role from this list will not remove that role when the server starts. That should be done manually.
+
+
+.. code-block:: rego
+
+    package policy
+
+    # Define roles
+    roles := ["read-only", "noc", "operator", "environment-admin", "environment-expert-admin"]
+
+    # Get the metadata for the specific endpoint that is called.
+    endpoint_data := data.endpoints[input.request.endpoint_id]
+
+    # Don't allow anything that is not explicitly allowed.
+    default allow := false
+    ...
+
+
+Enable the policy-engine authorization provider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Set the :inmanta.config:option:`server.authorization-provider` config option to ``policy-engine``.
+2. Modify the default policy at ``/etc/inmanta/authorization/policy.rego`` if desired. More information about writing your own access policy is available :ref:`here<authorization-writing-own-policy>`.
+3. If a 3rd party auth broker is used, it must be configured to correctly set all claims required by the access policy.
+4. Restart the inmanta server to activate the configuration changes.
+
+
+.. _authorization-writing-own-policy:
+
+Writing a custom access policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides some guidance on how to test/debug your own access policy.
+
+
+Troubleshooting: Policy engine fails to start
+"""""""""""""""""""""""""""""""""""""""""""""
+
+When the access policy contains a syntax error, the policy engine will fail to start and as such also the Inmanta server. In that case, the ``server.log`` file will indicate that the policy engine failed to start with a reference to the log file of the policy engine. The latter log file will contain more information about the specific issue. If required, the log level of the policy engine can be increased using the :inmanta.config:option:`policy_engine.log-level` config option.
```

Comment:
I hadn't noticed this in review. But why is this a separate option, rather than part of the generic logging config?